db/controller/controller.js

```diff
@@ -0,0 +1,10 @@
+const { getTopicsModel } = require("../models/newsModels");
+
+exports.getTopicsController = (req, res, next) => {
+  getTopicsModel()
+    .then((topics) => {
+      res.status(200).send({ topics });
```

Comment:
Great that you're sending the response on an object on an appropriate key 🔑

Reply:
Thanks!